pkgs/servers/bird/bird-lg.nix

```diff
@@ -0,0 +1,49 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+let
+  generic = { mod, vendorSha, runTests ? true }:
```

Comment:
It'd be nice if both binairies could end up in the same package. No idea what's the best approach to do this though.